easybuild/easyconfigs/q/QIIME/QIIME-1.9.1.eb

```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Conda'
+
+name = "QIIME"
+version = "1.9.1"
+variant = "Linux-x86_64"
+
+homepage = 'http://qiime.org/'
+description = """QIIME is an open-source bioinformatics pipeline for performing microbiome analysis from raw DNA sequencing data."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+requirements = "python=2.7 %(namelower)s=%(version)s matplotlib=1.4.3 mock nose"
```

Comment:
@pescobar why hardcoding the `2.7` here? The Anaconda version you're using already implies this?